### ISSUE TITLE
Goldleaf 0.8.0 crashes upon starting

### ISSUE THREAD

**Issue by TitoYisus** [XorTroll/Goldleaf#516]
So I openned the hbmenu and tried opening Goldleaf. But it just shows a black screen and sends me back to the hbmenu. I looked on the reports folder and found a fatal one and a couple of crashes of other tries to launch the program later.
[fatal_0x19C8C.log](https://github.com/XorTroll/Goldleaf/files/4405906/fatal_0x19C8C.log)
[crash_0x7A2450.log](https://github.com/XorTroll/Goldleaf/files/4405905/crash_0x7A2450.log)

I'm on Firmware 9.2.0 and Atmosphère 0.10.5-master-b27c7552

Any help?

**Comment by Enspiron** [comment]
sysdvr installed?

**Comment by TitoYisus** [comment]
> sysdvr installed?

I thought that may be the problem, but I waited por a reply first. Yes, I have SysDVR installed. Gonna unninstall and try again, thanks.

Just for curiosity, why does that happen?

**Comment by XorTroll** [comment]
USB stuff should only be used by one single process, and since both Goldleaf and SysDVR use it, Goldleaf behaves unexpectedly since when it starts up SysDVR is already using USB services.
TLDR: you can't have two homebrews using USB at the same time

**Comment by TitoYisus** [comment]
> USB stuff should only be used by one single process, and since both Goldleaf and SysDVR use it, Goldleaf behaves unexpectedly since when it starts up SysDVR is already using USB services.
> TLDR: you can't have two homebrews using USB at the same time

I see, thank you!!
